### ISSUE TITLE
merging with pmovil?

### ISSUE THREAD

**Issue by cdenneen** [pcfens/puppet-filebeat#6]
I see they have added CentOS and Fedora maybe try to combine these modules?

https://github.com/Pmovil/puppet-elastic_filebeat

**Comment by pcfens** [comment]
I'm not opposed to looking at it to see if it makes sense - merging would get a little complicated regarding namespace and all (does this one merge into Pmovil or does Pmovil merge into this one, or do we try and put them both somewhere else like puppet-community).

Other than the current duplication of efforts, is there anything in particular that one module is doing that you'd like to see in the other?

Do you mean the metadata.json file lists CentOS and Fedora support? I've been really bad about keeping the metadata.json file up to date. I'm adding the OSes I've tested on now. (@axier has been awesome and helping add Windows support too, but its not been fully tested yet).

**Comment by cdenneen** [comment]
I guess my biggest reason would be just for higher degree of collaboration,
better chance for approved modules, etc

For that reason definitely liking the idea puppet-community is trying to
accomplish.

CentOS and Windows support would be a definite + for me. Also implementing
options for topbeat and packetbeat would be a nice enhancement as well.

**Comment by pcfens** [comment]
If we focus on writing a high quality module then everyone wins regardless of whether or not the quality has been recognized by Puppet Labs, so I'm more concerned about writing a good module than I am about the labels attached to it. To that end, I'm happy to collaborate with anyone that also wants to produce a high quality module that does a good job of serving its purpose, whether its through regular contact or the occasional pull request.

CentOS is supported in the current release, I just didn't list every RH variant in the metadata file, so you should be ready to go if you're waiting on that. The next release should list more variants that I've tested on since the last release.

@dasrecht opened #2 to talk about adding packetbeat and topbeat support, so I'll keep that discussion there (the short version is I don't know enough about it to do it today, but the idea is intriguing).

**Comment by pcfens** [comment]
I'm going to go ahead and close this, but I'm happy to discuss the idea further with anyone that wants to.
